### ISSUE TITLE
Syntax highlighting in yaml files

### ISSUE THREAD

**Issue by benjaminbosko** [panoply/vscode-liquid#84]
I would like to have the same syntax highlighting in yaml files opened in VS Code as for example in json files.
![image](https://user-images.githubusercontent.com/8402823/155342478-324a9d8f-85d9-4cec-87e8-6e21956061ae.png)
![image](https://user-images.githubusercontent.com/8402823/155342172-d1031c81-a98d-4e0c-b982-0e37642e2c90.png)

Thank you!

**Comment by panoply** [comment]
🚢 Shipping in [v3.0.0](https://github.com/panoply/vscode-liquid/releases/tag/v3.0.0)

**Comment by benjaminbosko** [comment]
Thank you for the new release!
Is it possible to support yaml files without change its extension to `.yaml.liquid`? Unfortunately it is not an option for me.  
Moreover I have to use quotes in many situations for templates. Could you apply the syntax highlighting to quoted strings as well?  
![image](https://user-images.githubusercontent.com/8402823/192823022-c0678569-ef87-43e8-81cc-ce4f9d265b5d.png)

**Comment by panoply** [comment]
Hey @benjaminbosko,

So Yaml files will be highlighted using injections, so you don't need to use an extension - `file.yaml` will work no problems. I will have a look at support strings.

**Comment by panoply** [comment]
🚢 #103 

Injections / highlights and no suffix extension all working. Hope it helps!

**Comment by benjaminbosko** [comment]
Sorry, but I I see no changes at all. Am I doing something wrong?

2.3.0
![image](https://user-images.githubusercontent.com/8402823/192947861-8c197822-d417-456e-9166-cf9d64aef676.png)
![image](https://user-images.githubusercontent.com/8402823/192950177-11db253a-f86a-4f3f-98ba-2164c0ab79fd.png)
3.0.0
![image](https://user-images.githubusercontent.com/8402823/192947882-f436256f-a31b-4ba9-bc5c-ce2fb97118d7.png)
![image](https://user-images.githubusercontent.com/8402823/192950244-0ce4db49-68bf-4f00-bfcb-06ed5ae3bed9.png)
3.0.2
![image](https://user-images.githubusercontent.com/8402823/192947914-c45f8d41-6ab1-49e7-8ed9-2aa2476a1010.png)
![image](https://user-images.githubusercontent.com/8402823/192950278-c9033772-4c17-46f7-9cf5-3fb900eb38ac.png)
Disabled
![image](https://user-images.githubusercontent.com/8402823/192947946-3e187a0e-8285-4d7d-9d17-35958bf490ab.png)
![image](https://user-images.githubusercontent.com/8402823/192950326-8d51ac6f-fe91-4e47-a00f-f5617b3b9c38.png)

Moreover it used to work with JSON files, but not in the newer versions.

2.3.0
![image](https://user-images.githubusercontent.com/8402823/192949168-b35a0c31-a5a9-4d25-9a6e-0f732001bbe7.png)
3.0.2
![image](https://user-images.githubusercontent.com/8402823/192949137-cff9378b-1863-4e08-9f24-1e393f47ed1a.png)

**Comment by panoply** [comment]
I am unable to recreate. Testing YAML files all pass:

<img width="932" alt="Screenshot 2022-09-29 at 20 24 44" src="https://user-images.githubusercontent.com/7041324/193112871-658fdfc6-53ce-4523-9a71-bce4d90cdcb9.png">

Have you got any other extensions active?

> Moreover it used to work with JSON files, but not in the newer versions.

String encapsulated Liquid captures are not supported in JSON in v3.0 but create a new issue for that and I will look into it possibly reverting that change.

**Comment by benjaminbosko** [comment]
Thanks for your help, you were right! I disabled all my extension apart from Liquid and that way I figured out that the Jinja extension overwrote the behavior of Liquid. Now it is working fine:
![image](https://user-images.githubusercontent.com/8402823/193198640-2a8869e9-604f-402b-b114-d7b3675642ae.png)
However the syntax highlighting is not as advanced as in your case. I can live with that but may you know what configuration do I miss to have the same result as on your screenshot?
